pkg/scheduler/apis/config/validation/validation.go

```diff
@@ -97,10 +97,90 @@ var removedPluginsByVersion = []removedPlugins{
 	},
 	{
 		schemeGroupVersion: v1beta2.SchemeGroupVersion.String(),
-		plugins:            []string{"NodeLabel", "ServiceAffinity", "NodePreferAvoidPods"},
+		plugins: []string{
+			"NodeLabel",
+			"ServiceAffinity",
+			"NodePreferAvoidPods",
+			"NodeResourcesLeastAllocated",
+			"NodeResourcesMostAllocated",
+			"RequestedToCapacityRatio",
+		},
 	},
 }
 
+type conflictPlugin struct {
```

Comment:
we don't need this, just iterate over all extensions

Reply:
If pluginA implement `score` and `filter`, pluginB implement `score` and `filter` too, A and B only conflict in `score` stage, and pluginB should or not enabled in `filter` stage?

Reply:
This case doesn't exist right now, so we don't need to address it at the moment. Also, in the next release we are targeting to simplify plugin configuration where users will list plugins without specifying the exact extension points, see https://github.com/kubernetes/kubernetes/issues/102303